src/Faker/Provider/Miscellaneous.php

```diff
@@ -273,10 +300,21 @@ public static function languageCode()
 
     /**
      * @example 'EUR'
+     *
      * @link https://en.wikipedia.org/wiki/ISO_3166-1_alpha-2
      */
     public static function currencyCode()
     {
         return static::randomElement(static::$currencyCode);
     }
+
+    /**
+     * Returns an encoded Unicode Character between U+1F600 and U+1F637.
+     *
+     * @link https://en.wikipedia.org/wiki/Emoji#Unicode_blocks
+     */
+    public static function emoji()
+    {
+        return json_decode('"' . static::randomElement(static::$emoji) . '"');
```

Comment:
I did that in my original version 62b87b5de007df5dac40594df9258cbf397c4bbc , however the "\u{xxxx}" notation wasn't added to PHP until 7.0, so it failed the 5.x Travis builds.

After research this was apparently the best method to store unicode characters in PHP pre-7.0